task2/Task2TeamNull/point.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Xml.Serialization;
+
+namespace Task2TeamNull
+{
+    /// <summary>
+    /// Class to represent a point in two-dimensional space
+    /// </summary>
+
+    [Serializable]
+    public class Point
+
+    {
+        /// <summary>
+        /// Gets/Sets X coordinate of a point
```

Comment:
Just leave "X coordinate of a point", the same goes for Y and Z

Reply:
![This is a StyleCop requirement](https://user-images.githubusercontent.com/37864128/47594576-a6136700-d984-11e8-91a6-0d180378ffd8.png)
This is a StyleCop requirement